cartographer_ros/CMakeLists.txt

```diff
@@ -48,6 +48,11 @@ find_package(cartographer REQUIRED)
 include("${CARTOGRAPHER_CMAKE_DIR}/functions.cmake")
 option(BUILD_GRPC "build features that require Cartographer gRPC support" false)
 google_initialize_cartographer_project()
+# PCL requires C++14 since version 1.10.0 - override if needed.
+if (NOT ${PCL_VERSION} VERSION_LESS "1.10.0")
```

Comment:
`if (${PCL_VERSION} VERSION_GREATER_EQUAL "1.10.0")` would be nicer, but this doesn't work with CMake 3.5 from Ubuntu 16.